templates/package.json.template

```diff
@@ -34,9 +34,10 @@
     "dependencies": {
       "arguejs": "^0.2.3",
       "lodash": "^4.15.0",
+      "long": "^3.2.0",
```

Comment:
Quote from the docs:

If you need int64 support, explicitly require the long module somewhere in your project as it will be excluded otherwise. This assumes that a global require function is present that protobuf.js can call to obtain the long module.